glossary.yml

```diff
@@ -2855,6 +2855,22 @@
       not their data types or units).
 
 
+- slug: help_system
+  ref:
+    - documentation
+    - help_command
+    - help_function
```

Comment:
```suggestion
```

The `ref:` portion of these entries should point towards other existing entries, not simply provide synonyms or related terms because they are used to supply internal links at the bottom of the entry. We currently don't have any of these as entries.

Reply:
Thanks for pointing that out. I just noticed that when running into failing checks for PR #303. I've removed these for now.